docs/document/content/user-manual/shardingsphere-scaling/usage.cn.md

```diff
@@ -185,12 +207,25 @@ mysql> show scaling status 660152090995195904;
 | RUNNING                                           | 运行中                                                        |
 | EXECUTE_INVENTORY_TASK                            | 全量迁移中                                                     |
 | EXECUTE_INCREMENTAL_TASK                          | 增量迁移中                                                     |
-| ALMOST_FINISHED                                   | 基本完成                                                       |
-| FINISHED                                          | 已完成                                                         |
+| ALMOST_FINISHED                                   | 基本完成(偏内部使用的状态，不用特别在意这个状态。目前的设置逻辑：数据校验完成之后、切换配置之前)                                                       |
+| FINISHED                                          | 已完成（整个流程完成了，新规则已生效）                                                         |
 | PREPARING_FAILURE                                 | 准备阶段失败                                                    |
 | EXECUTE_INVENTORY_TASK_FAILURE                    | 全量迁移阶段失败                                                 |
 | EXECUTE_INCREMENTAL_TASK_FAILURE                  | 增量迁移阶段失败                                                 |
 
+如果`status`出现失败的情况，可以查看`Sharding-Proxy`的日志查看错误堆栈分析问题。
```

Comment:
We could just use `proxy` instead of `Sharding-Proxy`.

Reply:
get